controllers/controllers.js

```diff
@@ -14,6 +14,31 @@ exports.listApis = (req, res, next) => {
     }).catch(next)
 };
 
+exports.getArticles = (req, res, next) => {
+    return retrieveArticles()
+        .then((articles) => {
+            const lookup = {
+                9: 2,
+                3: 2,
+                5: 2,
+                6: 1,
+                1: 11
+            }
```

Comment:
This hard-coding of your lookup object means that this controller isn't dynamic. It is only built to work with your test data.

Try combining the queries from your comment_count and retrieve_article model into one using a join to create your comment_count column